dxr/static/templates/image_file.html

```diff
@@ -0,0 +1,40 @@
+{% extends "layout.html" %}
+{% from "partial/switch_tree.html" import tree_menu %}
+{% from "partial/no_results.html" import no_results %}
+
+{% block title %}{{ name }} - DXR{% endblock %}
+
+{% block head %}
+  <link rel="shortcut icon" href="{{ wwwroot }}/static/icons/mimetypes/{{ icon }}.png"/>
+{% endblock %}
+
+{% block site_css %}
+  {{ super() }}
+  <link href="{{ wwwroot }}/static/css/tree-selector.css" rel="stylesheet" type="text/css" media="screen" />
+  <link href="{{ wwwroot }}/static/css/image_file.css" rel="stylesheet" type="text/css" media="screen" />
+{% endblock %}
+
+{% block content %}
+<div id="content" class="content" data-no-results="{{ no_results() }}">
+  <div class="breadcrumbs">
```

Comment:
Yeah, that's definitely a good idea. I wasn't very familiar with Jinja templating at the time, but I've learned more about it working with it on some other projects recently.